nessie/sql_templates/create_boac_schema.template.sql

```diff
@@ -32,7 +32,7 @@ INTERLEAVED SORTKEY (user_id, course_id, assignment_id)
 AS (
     /*
      * Following Canvas code, in cases where multiple assignment overrides associate a student with an assignment,
-     * the override with the latest due date controls.
+     * then override with the latest due date controls.
```

Comment:
@raydavis "the" is actually intended here, though perhaps not felicitously phrased: "override" is a noun and "controls" a verb.

Reply:
How about a more upbeat, pro-social verb like "wins"?

Reply:
Yes, there can only be one winner.